### DISCUSSION TITLE
How to disable horizontal Scrollbar

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [radix-ui/primitives#2674] [Help] [answered]
In `ScrollArea` component when one of `Viewport` children's width is more than parent then it creates horizontal scrollbar. So how to force just vertical scrollbar?

**Comment by benoitgrelard** [comment, marked as answer]
If you don't want horizontal scroll, just don't render the `<ScrollArea.Scrollbar orientation="horizontal">`
